estate/models/estate_property_type.py

```diff
@@ -0,0 +1,20 @@
+from odoo import models, fields, api
+
+
+class PropertyType(models.Model):
+    _name = "estate.property.type"
+    _description = "the type of an estate"
+    _order = "sequence,name desc"
+
+    name = fields.Char("Type", required=True)
+    property_ids = fields.One2many("estate.property", "property_type_id")
+    sequence = fields.Integer()
+    offer_ids = fields.One2many("estate.property.offer", "property_type_id")
+    offer_count = fields.Integer(compute="_count_offer")
+
+    _sql_constraints = [("unique_property_tag", "UNIQUE(name)", "Tag name must be unique !")]
```

Comment:
```suggestion
    _sql_constraints = [("unique_property_type", "UNIQUE(name)", "Tag name must be unique !")]
```
 Constraint name `"unique_property_tag"` is misleading